frontend/src/core/models/task/task-model.ts

```diff
@@ -1,7 +1,22 @@
+import { TaskPriority } from './task-priority';
+import { TaskState } from './task-state';
+import { TaskType } from './task-type';
+
 export interface TaskModel {
-  currentUser?: number;
-  project?: [string, string, number];
-  issueType?: [string, string, number];
-  summary?: string;
-  description?: string;
+  id: number,
+  name: string,
+  description?: string,
+  summary: string,
+  taskState: TaskState,
```

Comment:
state. It belongs to 'Task', so there is no need to name the field 'taskState'

Reply:
Done